apps/s_time.c

```diff
@@ -260,9 +260,10 @@ int s_time_main(int argc, char **argv)
     printf
         ("\n\n%d connections in %.2fs; %.2f connections/user sec, bytes read %ld\n",
          nConn, totalTime, ((double)nConn / totalTime), bytes_read);
-    printf
-        ("%d connections in %ld real seconds, %ld bytes read per connection\n",
-         nConn, (long)time(NULL) - finishtime + maxtime, bytes_read / nConn);
+    if (nConn > 0)
+        printf
+            ("%d connections in %ld real seconds, %ld bytes read per connection\n",
+             nConn, (long)time(NULL) - finishtime + maxtime, bytes_read / nConn);
```

Comment:
Problem here is that both lines break 80 chars limit. For this reason I'd say that if would be more appropriate to re-format this as following
```
        printf("%d connections in %ld real seconds, "
               "%ld bytes read per connection\n",
               nConn, (long)time(NULL) - finishtime + maxtime,
               bytes_read / nConn);
```
On related note the printf above breaks 80-chars limit too. While we are at it, it wouldn't be inappropriate to re-format even that accordingly.

This comment naturally applies even to below modification.